docs/overview/00_gen.md

```diff
@@ -0,0 +1,24 @@
+
+# Olympus was created to be a better medium of exchange
+
+In 2021, Olympus DAO launched OHM to address shortcomings in stablecoins, Bitcoin, and Ethereum by introducing a new kind of decentralized, automated reserve currency backed by a diversified treasury. Today, the main objective is to provide utility and stability while being resistant to external market forces and speculative attacks.
+
+Here's how OHM addresses the shortcomings of stablecoins, Bitcoin, and Ethereum:
+
+Stablecoins: Stablecoins are typically pegged to a fiat currency, like the US Dollar, and rely on collateral to maintain their value. This exposes them to the risk of centralized control, regulatory crackdowns, and potential loss of backing. OHM, on the other hand, isn't pegged to a specific value. It's backed by a diversified treasury of various assets, making it more resistant to regulatory and market risks.
+
+Bitcoin: Bitcoin's monetary policy is hardcoded with a fixed issuance schedule and a maximum supply, which makes it predictable and resistant to malicious actors. However, it does not offer stability in terms of purchasing power due to its fixed-supply, which leads to hoarding and high volatility. OHM addresses this by implementing a flexible, automated monetary policy that can adapt to market conditions and maintain stability through mechanisms like Range-Bound Stability.
+
+Ethereum: Ethereum's monetary policy is subject to decentralized governance and is deflationary when fees outpace issuance. While it offers flexibility, it's still exposed to the risks of volatile market conditions. OHM's approach focuses on maintaining stability in purchasing power and offering a more predictable day-to-day store of value, thanks to its diversified treasury and unique economic mechanisms.
+
+By introducing an innovative monetary policy and a diversified treasury, OlympusDAO aims to create a more resilient reserve currency that can address the limitations of existing stablecoins, Bitcoin, and Ethereum, ultimately providing a predictable store of day-to-day value and a foundation for a new decentralized financial ecosystem.
+
+## Olympus has a rich history that bootstrapped the powerful treasury that powers the Range Bound Stable monetary policy today.
+
+From pioneering Protocol Owned Liquidity to delivering Bonding-as-a-Service to DeFi, Olympus has a past and present of innovation.
+
+To go down the rabbit hole and into the history books, visit Stablecoin Labs’ comprehensive course. The course covers the full timeline of Olympus, from genesis to early 2023 developments.
+
+###Olympus Scholars Program 
```

Comment:
```suggestion
### Olympus Scholars Program 
```
fix this header by adding a space
<img width="633" alt="image" src="https://user-images.githubusercontent.com/80423742/234569747-6844ee00-6d68-4e7c-aecc-f9123c2ba655.png">